packages/tome2/build.sh

```diff
@@ -0,0 +1,16 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/tome2/tome2
+TERMUX_PKG_DESCRIPTION="An open world roguelike adventure set in middle earth"
+TERMUX_PKG_LICENSE="custom"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_SRCURL=https://github.com/tome2/tome2.git
+TERMUX_PKG_VERSION=2.4.0-ah-2022.02.24
+TERMUX_PKG_GIT_BRANCH=master
+_COMMIT=d25bdae09bffea46ac54e51b99b2c166d9be7db8
```

Comment:
`_COMMIT` var won't make repo checkout to it. Use something like:
https://github.com/termux/termux-packages/blob/40efdad80188f0a7a170a1a363274227f44ce181/disabled-packages/librusty-v8/build.sh#L14-L22

Reply:
Oh dang! That's not automagic huh? Thanks for the heads up! (And oooops)

I pushed a new version that grabs the commit this time, and verifies the checkout succeeded to move HEAD where it needs to go before building. How does this look..?

Reply:
Looks nice!

Just one more suggestion. I think it would be better to use commit date as version, since it isn't really `2.4.0-ah` version.

Reply:
Sounds good to me! I think I'll squash all the commits together to clean things up a little too for the PR~

Reply:
https://github.com/termux/termux-packages/pull/10923#issuecomment-1149479758

oh, ha